webrtc-stats.html

```diff
@@ -1577,6 +1579,33 @@ <h2>
                   the configuration is considered power efficient.
                 </p>
               </dd>
+              <dt>
+                <dfn>totalAssemblyTime</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  Only [= map/exist =]s for video. The sum of the time, in seconds, each video frame takes
+                  from the time the first RTP packet is received (reception timestamp) and to the time
+                  the last RTP packet of a frame is received.
+                </p>
+                <p>
+                  Given the complexities involved, the time of arrival or the reception timestamp is measured as close to the network layer as possible.
+                  This metric is not incremented for frames that are not decoded, i.e., framesDropped, partialFramesLost or frames that fail decoding
+                  for other reasons (if any). Only incremented for frames consisting of more than one RTP packet. The average frame assembly time can
+                  be calculated by dividing the totalAssemblyTime with framesAssembledFromMultiplePackets.
```

Comment:
Do we need `framesAssembledFromMultiplePackets` wouldnt framesDecoded suffice?

Reply:
If majority of frames fit into 1 packet, frameAssemblyTime will be zero, and those drown out the assembly-time signal.

Reply:
Got the answer -- not all the framesDecoded are counted in the assembly time

Reply:
No because framesDecoded is after decoding and also includes frames that were single packet

Reply:
in practice this only happens with 320x180 thumbnails but it does happen sadly so we needed the separate counter.